app/views/core/CreateAccountModal/teacher/SchoolInfoPanel.coffee

```diff
@@ -29,6 +29,10 @@ SchoolInfoPanel =
       countriesList: countryList.getNames()
       usaStates: new UsaStates().states
       usaStatesAbbreviations: new UsaStates().arrayOf('abbreviations')
+      countryMap:
+        "Hong Kong": "Hong Kong, China",
```

Comment:
This will eventually be reported in the "country" field of the trial request, right?  If so this is going to break quite a few downstream integrations that we have based on this.  We report country to quite a few of our 3rd party tools and many of them have strict requirements around the format of country.

Most of these tools are only going to be used for US users, so we could filter out these values and replace them with `China` in our 3rd party integrations but that would be a hack that will surely cause us problems at some point in the future.

On this page we also have a "state" field.  What if we changed that to a "state / region" field and replaced that list with a list of China regions we wanted when the user selects China?  We could also only make this list modification in China.  It's still a hack and I'd guess it will still cause problems in the future, but at least it's reducing the potential impact.

Reply:
> This will eventually be reported in the "country" field of the trial request, right? 
I think this just changes the display value, not the actual saved value, right?
```
            option(v-for="country in countries" v-bind:value="country")
```
Shouldn't this indicate that the value is still `country`, even though the text shown has been updated?

> On this page we also have a "state" field. What if we changed that to a "state / region" field ...
The point is that what some users may think of as a separate country, like Macao, to China are separate regions of China, so saying "Country / Region" is flexible there. Otherwise, we are taking too strong a stance by putting these places in a list where they don't expect to find them (as listing parallel to other countries, but clarifying that they're part of China, seems to be the more standard approach). We can't do this just in China, it has to be in other regions as well.

Reply:
Ah I didn't realize that we'd already had these in the country list!  Great.  LGTM!